models/Team.js

```diff
@@ -1,10 +1,11 @@
 const mongoose = require('mongoose');
 const Schema = mongoose.Schema;
+const playerSchema = require('./Player').schema;
 
 const teamSchema = new Schema({
   name: String,
-	owner: String,
-	players: [String]
+	owner: playerSchema,
```

Comment:
Owner is a User schema, that way we can look up based off what User is logged in.